terraform/tls-web-application/README.md

```diff
@@ -0,0 +1,59 @@
+# DNSimple TLS & Terraform
+
+An example library using Terraform to configure a DNSimple registered domain with TLS using Let's Encrypt.
+
+The demo web application is built using vite + react. The web application build is copied into the docker image defined in `terraform/main.tf` along with the resulting SSL/TLS certificate data and required nginx configurations.
+
+## Requirements
+
+- Docker
+- NodeJS
+- Terraform
+
+## DevContainer
+
+The provided `.devcontainer` configures Debian Bullseye with:
+
+- common-utils
+- docker-outside-of-docker
+- node
+- terraform
+
+## Setup
+
+### Build Web Application
+
+Requires NodeJS + preferred package manager (examples using yarn)
+
+1. `cd react-app`
+2. `yarn && yarn build`
```

Comment:
What do you think about dropping the react-app in favour of a static page? My thinking is that we reduce the number of requirements and external dependencies by removing the need for Node, and we can still showcase the utility of the demo. WDYT?

Reply:
Agreed, I have moved the static assets into the `app` folder and removed the build requirement / node as a setup dependency.